modules/components/pages/processors/benchmark.adoc

```diff
@@ -0,0 +1,60 @@
+= benchmark
+// tag::single-source[]
+:type: processor
+:page-beta: true
+:categories: ["Utility"]
+
+component_type_dropdown::[]
+
+Logs throughput statistics for messages that pass through this processor, and provides a summary of those statistics over the lifetime of the processor.
```

Comment:
```suggestion
Logs message throughput statistics for this processor and summarizes them over its lifetime.
```

messages thought this processor seems redudant. Consider this simplification

Reply:
Generally, I'd avoid long noun strings. I've made some other updates. Thanks for the feedback.

Reply:
Nice! I like the update